src/Appwrite/Platform/Tasks/Install.php

```diff
@@ -50,7 +53,6 @@ public function action(string $httpPort, string $httpsPort, string $organization
          * 6. Run data migration
          */
         $config = Config::getParam('variables');
-        $path = '/usr/src/code/appwrite';
         $defaultHTTPPort = '80';
```

Comment:
Can we also add a confirmation dialog of an existing setup is detected. This way we can ask the developer to confirm if they want to overwrite it and understand the implications of it.

Reply:
How's this?

install:

<img width="483" alt="image" src="https://github.com/appwrite/appwrite/assets/1477010/9deb7d3e-bf9f-4f92-be98-a47252b1de22">

upgrade:

<img width="541" alt="image" src="https://github.com/appwrite/appwrite/assets/1477010/0fe098fb-558a-4fd6-a73c-d71927440d26">

Reply:
I like it.